test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua

```diff
@@ -231,25 +285,38 @@ local function DisallowedRPC(RPC, params, interface, self)
   commonDefects.delayedExp()
 end
 
+--[[ @ptu: Policy table update
+--! @parameters:
+--! pUpdateFunction - update table for PTU
+--! self - test object
+--! @return: none
+--]]
+local function ptu(pUpdateFunction, self)
+  EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UP_TO_DATE" })
+  commonDefects.ptu(pUpdateFunction, self)
+  CheckCuttingUnknowValues(1, self)
+end
+
 --[[ Scenario ]]
 runner.Title("Preconditions")
 runner.Step("Clean environment", commonDefects.preconditions)
 runner.Step("Start SDL, HMI, connect Mobile, start Session", commonDefects.start)
 
 runner.Title("Test")
-runner.Step("App registration, PTU with unknown API", CheckCuttingUnknowValues, { ptuUpdateFuncRPC })
+runner.Step("App registration, PTU with unknown API", rai_with_OnPermissionChange, { ptuUpdateFuncRPC })
 runner.Step("Check applying of PT by processing SendLocation", SuccessfulProcessingRPC,
-  { "SendLocation", SendLocationParams, "Navigation" })
-runner.Step("Unregister application", commonDefects.unregisterApp)
+  { "SendLocation", SendLocationParams, "Navigation", {} })
+runner.Step("Trigger PTU from HMI", triggerPTU)
 
-runner.Step("App registration, PTU with unknown parameters", CheckCuttingUnknowValues, { ptuUpdateFuncParams })
+runner.Step("PTU with unknown parameters", ptu, { ptuUpdateFuncParams })
 runner.Step("Check applying of PT by processing GetVehicleData", SuccessfulProcessingRPC,
-  { "GetVehicleData", { gps = true }, "VehicleInfo" })
+  { "GetVehicleData", { gps = true }, "VehicleInfo", { gps = gpsDataResponse } })
 runner.Step("Check applying of PT by processing SubscribeVehicleData", DisallowedRPC,
   { "SubscribeVehicleData", { gps = true }, "VehicleInfo" })
 
 runner.Step("Remove Snapshot and trigger PTU, check new created PTS", removeSnapshotAndTriggerPTUFromHMI)
-runner.Step("Invalid_PTU_after_cutting_of_unknown_values", commonDefects.unsuccessfulPTU, { ptuUpdateFuncNotValid })
+runner.Step("Invalid_PTU_after_cutting_of_unknown_values", commonDefects.unsuccessfulPTU,
```

Comment:
```suggestion
runner.Step("Invalid_PTU_after_cutting_off_unknown_values", commonDefects.unsuccessfulPTU,
```

Reply:
@jacobkeeler Please find update in d45c6c1